src/mpyl/utilities/repo/__init__.py

```diff
@@ -231,32 +228,21 @@ def changes_in_branch_including_local(self) -> Changeset:
             untracked_files=set(self._repo.untracked_files),
         )
 
-    def changes_in_tagged_commit(self, current_tag: str) -> Changeset:
-        current_revision = self._repo.head.commit
-        curr_rev_tag = self._repo.git.tag(current_revision, points_at=True)
-        logging.debug(f"Current revision: {current_revision} tag: {curr_rev_tag}")
-
-        if curr_rev_tag != current_tag:
-            logging.error(f"HEAD is at {curr_rev_tag} not at expected `{current_tag}`")
-            return Changeset.empty(self.get_sha)
-
-        return self.changes_in_merge_commit()
+    def changes_in_tagged_commit(self, tag: str) -> Changeset:
+        head_commit = self._repo.head.commit
+        tag_commit = self._repo.tag(f"refs/tags/{tag}").commit
```

Comment:
Maybe turn them into a string right away since I think you only use them as strings? (except for the if which will still work anyway I guess)

Reply:
Actually, I just remembered that [f-strings automatically convert objects to strings](https://peps.python.org/pep-0498/#specification) so there's no need to manually do `str(...)` anyway. I'll clean it up 👍